Lanchonete/Lanchonete/ Features/Confirmation/Presentation/ConfirmationViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ConfirmationViewController.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 1/25/21.
+//
+
+import UIKit
+
+final class ConfirmationViewController: UIViewController {
```

Comment:
This view controller does not implement a contract with the Presenter.
It would be very useful to create a protocol for it so you can spy your presenter in your unit tests and abstract the view to inject it in your presenter.
I can help you with this if you want :)